client/devdocs/design/search-collection.jsx

```diff
@@ -61,7 +62,10 @@ export default React.createClass( {
 		searchString = example.type.displayName;
 
 		if ( this.props.component ) {
-			return example.type.displayName.toLowerCase() !== this.props.component.replace( /-([a-z])/g, '$1' );
+			const exampleName = invoke( example, 'type.displayName.toLowerCase' );
+			const componentName = invoke( this, 'props.component.replace', /-([a-z])/g, '$1' );
```

Comment:
there's also `result()`, which works like `get` but invokes the property if it's a function

seriously though, **lodash** + **dash** + **alfred** is amazing. I have spent lots of time scanning through the library just because it's cool and because I think, "hmmm…I bet this already exists somewhere" and Dash has everything organized by category: _Array_, _Object_, _Language_, _Util_, etc…